### PR TITLE
Implemented picker

### PR DESCRIPTION
**Pull request by ratthompson**
Implemented picker for a trainer's batches

### CODE REVIEW

File: batches/batch.service.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import Batch from './batch';
+
+class BatchService {
+    private URI: string;
+    constructor() {
+        this.URI = /*process.env.CALIBER_URI*/'https://aosczl5fvf.execute-api.us-west-2.amazonaws.com/default' + '/batches';
+    }
+
+    getBatchesByTrainerEmail(): Promise<Batch[]> {
```
Comment (jedawson) [inline batches/batch.service.ts]: would it be better to declare a trainer email outside this function and then pass in the trainer email into this function as an argument?

Comment (jedawson) [reply batches/batch.service.ts]: nevermind, the trainer email is not in the store.

File: test/batches/batches.component.test.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import 'react-native';
+import 'jest-enzyme';
+import '@testing-library/jest-dom';
+import Enzyme, {shallow} from 'enzyme';
+
+import BatchesComponent from '../../batches/batches.component';
+
+describe('BatchesComponent Test(s)', () => {
+    test('the component did mount', () => {
+});shallow()<BatvchesComponent /> const wrapper = Enzyme.mount(
```
Comment (jedawson) [inline test/batches/batches.component.test.ts]: should it be BatchesComponent?
```suggestion
});shallow()<BatchesComponent /> const wrapper = Enzyme.mount(
```

**Review by jedawson — COMMENTED** [review]
I haven't created a picker, but a nested iife inside of an iife seems bizarre.

File: batches/batch.service.ts (hunk 1 above)
Comment (jedawson) [reply batches/batch.service.ts]: let trainerEmail = 'mock1027.employee74df14df-5842-4811-a57c-be9836537a40@mock.com'
getBatchesByTrainerEmail(trainerEmail): Promise<Batch[]> {

and then 
return axios.get(this.URI, {params: {trainerEmail: trainerEmail}})
```suggestion
    getBatchesByTrainerEmail(trainerEmail): Promise<Batch[]> {
```

File: batches/batch.service.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        console.log('Batch Service: getBatchesByTrainerEmail');
+        return axios.get(this.URI, {params: {trainerEmail: 'mock1027.employee74df14df-5842-4811-a57c-be9836537a40@mock.com'}})
```
Comment (jedawson) [inline batches/batch.service.ts]:
```suggestion
        return axios.get(this.URI, {params: {trainerEmail: trainerEmail}})
```

**Review by jedawson — COMMENTED** [review]
looks good to me

**Comment by Tngtater9** [comment]
I know some of the code smells are from redux but if you could take care of the ones in the batches component I'll approve

**Review by Tngtater9 — APPROVED** [review]
Looks good I'm assuming you kept useNavigation because you plan to use it later

**Comment by ratthompson** [comment]
Yup, we'll need to link to batch info once that's implemented

**Pull request merged**